src/language-js/utils.js

```diff
@@ -352,15 +349,14 @@ function isTheOnlyJsxElementInMarkdown(options, path) {
  * @returns {boolean}
  */
 function isMemberExpressionChain(node) {
```

Comment:
Do we want to consider `TSNonNullExpression` part of "member expression chain" in this function?
E.g.: `a.b!.c`
If we do, is a simple expression like `a!`? I guess no. But what about `a.b.c!` (a chain _wrapped_ in a non-null assertion)?

Reply:
This function is used only for assignments, so you can ignore my comment above. I've solved that in #10222.